tests/scripts/basic-build-test.sh

```diff
@@ -76,7 +76,7 @@ make -j
 TEST_OUTPUT=out_${PPID}
```

Comment:
Commit message: I don't understand why because with no longer enable MBEDTLS_ENTROPY_FORCE_SHA256 the seedfile size have to be increased. Usually when you remove something you don't need more resources.  I understand it is just a lack of knowledge on my side. Thus fine by me if it is not really possible to improve it but if you think about some improvements, please do it.

Reply:
Yes, that one does require understanding several parts of the library.

* `seedfile` stores entropy across sessions. It is only used when `MBEDTLS_ENTROPY_NV_SEED` is enabled. When enabled, it must be present when the library starts (more precisely, at a call to `mbedtls_entropy_init()`, which most real-world programs call, as well as `psa_crypto_init()` and some test suites — but there are test suites that use a fake RNG and so don't call `mbedtls_entropy_init()`).
* The size of the seed file must be at least the size of the entropy pool, otherwise `mbedtls_emtropy_init()` fails. A (successful) call to `mbedtls_entropy_init()` rewrites the seed file to exactly the size of the entropy pool. (This can matter if you run successive tests with different sizes for the entropy pool.)
* The size of the entropy pool is the size of the hash that's used to accumulate entropy: 32 bytes for SHA-256, 64 bytes for SHA-512.
* The hash used to accumulate entropy is SHA-512 if `defined(MBEDTLS_SHA512_C) && !defined(MBEDTLS_ENTROPY_FORCE_SHA256)` and SHA-256 otherwise.

Now for some history:

* In the past, `config.py full` enabled `MBEDTLS_ENTROPY_FORCE_SHA256`, because it enables boolean options unless explicitly excluded and this option was not excluded. As a consequence, we created a 32-byte seedfile.
* Relatively recently, we removed `MBEDTLS_ENTROPY_FORCE_SHA256` from `config.py full`, because it's a booleam setting, not an optional component: you aren't including more code when the option is on, you're using a different branch in some parts of the code.
* So now the seedfile size should be 64 bytes. We updated `all.sh` accordingly a while ago, but didn't notice that `basic-build-test.sh` also needed to be updated because that script didn't properly report failures.

Reply:
Thanks for this comprehensive explanation. My understanding that we remove something was wrong, we rather changed the way things are done (different branches).

Reply:
I added more information to the commit message in #3181. It was obvious to me because I happened to be familiar with all the steps in that sordid tale, but the commit message is for the poor (or happy?) soul who isn't.